internal/bminventory/inventory.go

```diff
@@ -114,10 +116,41 @@ const ignitionConfigFormat = `{
       "path": "/etc/motd",
       "mode": 644,
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
-    }]
+	}]
```

Comment:
?

Reply:
Thank you. I will pay more attention to such auto-indentation caused by my editor.